app/src/main/java/com/sun/cooktails_app/ui/splash/SplashActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.sun.cooktails_app.ui.splash
+
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import androidx.appcompat.app.AppCompatActivity
+import com.sun.cooktails_app.R
+import com.sun.cooktails_app.ui.main.MainActivity
+
+@Suppress("DEPRECATION")
+class SplashActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash)
+        Handler().postDelayed({
+            val intent = Intent(this, MainActivity::class.java)
```

Comment:
https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md#25-arguments-in-fragments-and-activities